src/main/java/DukeBot/Duke.java

```diff
@@ -0,0 +1,49 @@
+package DukeBot;
+
+import DukeBot.command.Command;
+
+/**
+ * Encapsulates the Duke class.
+ */
+public class Duke {
```

Comment:
Having renamed your bot to Thesh, would it be better to rename your classes, `Duke` and `DukeException` to `Thesh` and `TheshException` as well for sake of clarity?